platform/core/src/core.rs

```diff
@@ -84,3 +111,48 @@ pub fn load_feature_categories(
         &core.language_code,
     )
 }
+
+// App events
+
+pub fn app_did_become_inactive() -> Result<(), CoreFailure> {
+    let mut instance = get_instance()?;
+    let session = instance
+        .user_session
+        .get_mut()
+        .map_err(|err| CoreFailure::failed_to_acess_user_usession(err.to_string()))?;
```

Comment:
`failed_to_acess_user_usession` has an extra `u` in the last word. It should be `failed_to_acess_user_session`

Reply:
mea culpa :)